main/opcodes.zkasm

```diff
@@ -932,10 +945,14 @@ opEXTCODECOPY:
     $ => E          :MLOAD(SP)   ; bytes to read from bytecode
     D               :MSTORE(lastMemOffset)
     E               :MSTORE(lastMemLength)
-    
-    ; Check counters
-    %MAX_CNT_BINARY - CNT_BINARY - 2*E :JMPN(outOfCounters)
-    %MAX_CNT_MEM_ALIGN - CNT_MEM_ALIGN - 2*E :JMPN(outOfCounters)
+
+    ;Check counters should be before than do all the operations! TODO
```

Comment:
I would open an issue with this topic and it will be reviewed once the code is finished.
I suspect changes that will modify `zk-counters` will be done in the coming PRs.
Agree that checks must happen before any instruction that consume `zk-counters` is executed